### PR TITLE
Release: 0.2.0

### PR DESCRIPTION
**Pull request by cojack**
Hi guys,
So we're planing to release new version, feel free to suggest anything, Im open.
I'm really nervous about that release because I don't wanna to fword anything up, and I know how many ppl are using this in their production systems, so I'm counting on every of you to help bring this up.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ language: node_js
 node_js:
   - stable
   - 8
-  - 6
```
Comment (MTschannett) [inline .travis.yml]: Do we really drop support for this?

Comment (NoNameProvided) [reply .travis.yml]: We do not. Let's re-add that.

**Comment by NoNameProvided** [comment]
> I'm really nervous about that release 

Tests are passing, so don't be nervous. 😄  

I have a few dev ops thing in mind, eg.:
- git history is not too clean tho, we should rebase PRs before merging or squash for smaller changes
- changelog how-to etc, 

but we can talk about these after release. I think nothing blocks the release of this tomorrow.

File: src/TransformOperationExecutor.ts [hunk 2]
```diff
@@ -14,33 +15,52 @@ export class TransformOperationExecutor {
...
 
-    transform(source: Object|Object[]|any,
-              value: Object|Object[]|any,
-              targetType: Function,
-              arrayType: Function,
-              isMap: boolean,
-              level: number = 0) {
+    transform(source: Object | Object[] | any,
+        value: Object | Object[] | any,
+        targetType: Function | TypeMetadata,
```
Comment (NoNameProvided) [inline src/TransformOperationExecutor.ts]: I am not happy about this overloading, this function is way bigger and messier than it should be, overloading I just add more to the confusion, but we can fix this after release.

File: .travis.yml (hunk 1 above)
Comment (cojack) [reply .travis.yml]: @NoNameProvided serious? Even angular drop node 6 for development, do we really need that? Im against.

**Comment by cojack** [comment]
Guys, I'm a bit busy, so don't look at me if you wanna push it forward, just let the things happen.

File: .travis.yml (hunk 1 above)
Comment (MTschannett) [reply .travis.yml]: I also read that express dropped support if I recognize correctly

Comment (NoNameProvided) [reply .travis.yml]: I can be bought in on dropping node 6, but then just go with `stable` and `lts/*` instead not? Anyway we can do this after release.

**Comment by NoNameProvided** [comment]
So I will be the one who needs to make the changelog? I will do no develop, but @cojack and @MTschannett don't hesitate to add update PRs for it if I miss something.

**Pull request merged**